src/pages/kimjihun/Main/Main.js

```diff
@@ -1,13 +1,310 @@
 import React from 'react';
+import Nav from '../../../components/Nav/Nav';
+import { withRouter } from 'react-router-dom';
+import CommentList from './CommentList';
 
+import './Main.scss';
 class MainHun extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      comment: '',
+      commentList: [],
+    };
+  }
+
+  handleComment = e => {
+    this.setState({
+      comment: e.target.value,
+    });
+  };
+
+  handleList = () => {
+    if (this.state.comment.length === 0) {
+      return;
+    }
+
+    const newComment = {
+      comment: this.state.comment,
+      id: Date.now(),
+    };
+
+    this.setState(state => ({
+      commentList: state.commentList.concat(newComment),
+      comment: '',
+    }));
+  };
+
   render() {
     return (
-      <div>
-        <p>명성</p>
+      <div className="MainHun">
```

Comment:
- 소문자로 시작해주세요! 대문자는 컴포넌트를 네이밍 하기 위한 컨벤션 입니다!
```suggestion
      <div className="mainHun">
```